src/styles/base/index.scss

```diff
@@ -1,20 +1,21 @@
-html {
-  box-sizing: border-box;
-}
-
-nav {
-  background-color: #313030;
-}
-
 body {
   font-family: 'Open Sans', sans-serif;
   margin:0; 
 }
 
-li{
+a {
   list-style-type: none;
 }
 
+a:hover {
```

Comment:
move this hover styles to your module, this hover state belongs only to your `.nav-item` item,